src/acl.c

```diff
@@ -220,6 +220,14 @@ uint64_t ACLGetCommandCategoryFlagByName(const char *name) {
     return 0; /* No match. */
 }
 
+/* Method for seraching for a user within a list of user definitions. The
+ * list contains an array of user arguments, and we are only
+ * searching the first argument, the username, for a match. */
+int ACLListMatchLoadedUser(void *a, void *b) {
```

Comment:
The names a and b are a little misleading if they're not of the same type.

Reply:
Sure, I'll fix it, still need to address Oran's comment.